crypto-serde/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "crypto-serde"
 description = "Serde serializer/deserializer helpers used by the RustCrypto project"
-version = "0.0.0"
+version = "0.1.0"
```

Not ready to cut a release yet
```suggestion
version = "0.0.0"
```